src/main/java/at/hannibal2/skyhanni/config/features/inventory/InventoryConfig.java

```diff
@@ -212,6 +213,13 @@ public String toString() {
     @FeatureToggle
     public boolean highlightWidgets = true;
 
+    @Expose
+    @ConfigOption(name = "SkyBlock XP Bar", desc = "Replaces the vanilla XP bar with a SkyBlock XP bar.")
+    @SearchTag("skyblockxp")
+    @ConfigEditorBoolean
+    @FeatureToggle
+    public boolean skyblockXpBar = false;
```

Comment:
this is not inventory related, this is misc. only the data fetch is from inventory, but the user facing change (the change of numbver shown in the xp bar) is not related to the inventory. please move the config over from inv -> misc